README.md

```diff
@@ -16,6 +16,46 @@ The following tags are officially supported:
 
 https://hub.docker.com/r/liquibase/liquibase/tags
 
+### liquibase alpine
+
+The `liquibase:<version>-alpine` tag is a slimmed-down version of the Liquibase Docker container. It is designed to be lightweight and have a smaller footprint, making it suitable for environments with limited resources or when only the essential functionality is required.
+
+#### Functionality and Purpose
+
+The `liquibase:<version>-alpine` container provides the core functionality of Liquibase, which includes database change management and version control. It allows you to define and manage database schemas, apply and roll back changes, and track the evolution of your database over time.
+
+#### Usage and Prerequisites
+
+To use the `liquibase:<version>-alpine` container, you need to have Docker installed on your system. Please refer to the official Docker documentation for instructions on how to install Docker: [https://docs.docker.com/get-docker/](https://docs.docker.com/get-docker/)
+
+#### Examples and Usage Scenarios
+
+Here are some examples to demonstrate the capabilities of `liquibase-slim`:
+
+1. **Initializing a new Liquibase project**:
+
+```shell
+docker run --rm -v $(pwd):/liquibase/changelog liquibase/liquibase:<version>-alpine \
+  --classpath=/liquibase/changelog \
+  --changeLogFile=changelog.xml \
+  --url="jdbc:postgresql://localhost:5432/mydb" \
+  --username=myuser \
+  --password=mypassword \
+  generateChangeLog
+```
+
+2. **Applying database changes**:
+
+```shell
+docker run --rm -v $(pwd):/liquibase/changelog liquibase/liquibase:<version>-alpine \
+  --classpath=/liquibase/changelog \
+  --changeLogFile=changelog.xml \
+  --url="jdbc:postgresql://localhost:5432/mydb" \
+  --username=myuser \
+  --password=mypassword \
+  update
+```
+
```

Comment:
And what about?

```
#### Functionality and Purpose

The `liquibase:<version>-alpine` container provides the core functionality of Liquibase, which includes database change management and version control. It allows you to define and manage database schemas, apply and roll back changes, and track the evolution of your database over time.

#### Usage and Prerequisites

To use the `liquibase:<version>-alpine` container, you need to have Docker installed on your system. Please refer to the official Docker documentation for instructions on how to install Docker: [https://docs.docker.com/get-docker/](https://docs.docker.com/get-docker/)
```

I would delete it as well.

Reply:
agreed.  Everything about it is the same except for the base image.  I like what eclipse temurin did:
https://hub.docker.com/_/eclipse-temurin

It might also be good to call out (or provide examples) of extending the alpine dockerfile like we did with the ubuntu-based dockerfile for things like native executors, aws cli, etc.  Thoughts?

Reply:
Let me re-write https://hub.docker.com/_/eclipse-temurin and add some alpine examples